subscription-engine/kafka-ws/app/Kafka/KafkaConsumer.js

```diff
@@ -13,8 +14,10 @@ class KafkaConsumer {
   * Instance KafkaConsumer
   */
   constructor() {
+    this.consumerConfig = ConfigManager.getConfig(KAFKA_WS_CONFIG_LABEL).consumer;
+
     logger.debug('constructor: Instance KafkaConsumer');
-    this.consumer = new Consumer(consumerConfig);
+    this.consumer = new Consumer(this.consumerConfig);
```

Comment:
I think we have a problem here. Look at https://github.com/dojot/dojot-microservice-sdk-js/blob/68950d6d71b0a3b16ae5cb5f560fdbab2dbbf7a9/examples/consumer/sample.js#L13 . The first parameter is consumer settings, but the second one is topic settings. We are passing `consumer.auto.offset.reset=largest` as a consumer property, but it is a topic property.

Other services have this same problem.